src/coreclr/jit/codegenarm64test.cpp

```diff
@@ -4772,6 +4772,28 @@ void CodeGen::genArm64EmitterUnitTestsSve()
     theEmitter->emitIns_R_R_R_R(INS_sve_cmpne, EA_SCALABLE, REG_P3, REG_P1, REG_V15, REG_V20,
                                 INS_OPTS_SCALABLE_H); // CMPNE   <Pd>.<T>, <Pg>/Z, <Zn>.<T>, <Zm>.<T>
 
+    // IF_SVE_CX_4A_A
+    theEmitter->emitIns_R_R_R_R(INS_sve_cmpeq, EA_SCALABLE, REG_P15, REG_P7, REG_V31, REG_V3,
+                                INS_OPTS_SCALABLE_WIDE_B); /* CMPEQ   <Pd>.<T>, <Pg>/Z, <Zn>.<T>, <Zm>.D */
```

Comment:
This should wait until https://github.com/dotnet/runtime/pull/96692 as `INS_OPTS_SCALABLE_WIDE_B` is moving to a new enum.

Reply:
Done!